src/components/overview/dashboardInsights.tsx

```diff
@@ -9,56 +10,73 @@ interface DashboardInsightsProps {
+                  : "-"
+                : ""}{" "}
+              {overview?.weekGrowth.toFixed(2)}% all time growth during last
+              week
+            </p>
+          </CardContent>
+        </Card>
+      )}
 
-      <Card>
-        <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
-          <CardTitle className="text-sm font-medium">
-            Impact this week
-          </CardTitle>
-          <Icons.gitBranch></Icons.gitBranch>
-        </CardHeader>
-        <CardContent>
-          <div className="text-2xl font-bold">
-            {overview?.weekScore.toFixed(2)}
-          </div>
-          <p className="pt-1 text-xs text-muted-foreground">
-            {overview?.wowScore ? (overview?.wowScore > 0 ? "+" : "-") : ""}{" "}
-            {overview?.wowScore.toFixed(2)}% compared to last week
-          </p>
-        </CardContent>
-      </Card>
+      {overview?.totalScore == null ? (
+        <CardSkeleton></CardSkeleton>
+      ) : (
+        <Card>
+          <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+            <CardTitle className="text-sm font-medium">
+              Impact this week
+            </CardTitle>
+            <Icons.gitBranch></Icons.gitBranch>
+          </CardHeader>
+          <CardContent>
+            <div className="text-2xl font-bold">
+              {overview?.weekScore.toFixed(2)}
+            </div>
+            <p className="pt-1 text-xs text-muted-foreground">
+              {overview?.wowScore ? (overview?.wowScore > 0 ? "+" : "-") : ""}{" "}
+              {overview?.wowScore.toFixed(2)}% compared to last week
+            </p>
+          </CardContent>
+        </Card>
+      )}
 
-      <Card>
-        <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
-          <CardTitle className="text-sm font-medium">
-            Total contributors
-          </CardTitle>
-          <Icons.users></Icons.users>
-        </CardHeader>
-        <CardContent>
-          <div className="text-2xl font-bold">
-            {overview?.totalContributors}
-          </div>
-          <p className="pt-1 text-xs text-muted-foreground">
-            Contributors on team
-          </p>
-        </CardContent>
-      </Card>
+      {overview?.totalScore == null ? (
+        <CardSkeleton></CardSkeleton>
+      ) : (
+        <Card>
+          <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+            <CardTitle className="text-sm font-medium">
+              Total contributors
+            </CardTitle>
+            <Icons.users></Icons.users>
+          </CardHeader>
+          <CardContent>
+            <div className="text-2xl font-bold">
+              {overview?.totalContributors}
+            </div>
+            <p className="pt-1 text-xs text-muted-foreground">
+              Contributors on team
+            </p>
+          </CardContent>
+        </Card>
+      )}
```

Comment:
I think we should move these cards to a separate component, the same way we did on the teams/[teamid]/page.tsx
It's getting super crowded and confusing on this file.

Ideally, we would also have a cardSkeleton component that reflects those 3 cards as well, so in this page it would look something like this: 

{loading ?
<AccountOverviewSkeletonCards>
: 
<AccountOverviewCards>}